packages/console/src/components/ItemPreview/index.tsx

```diff
@@ -1,19 +1,32 @@
 import React, { ReactNode } from 'react';
+import { Link, To } from 'react-router-dom';
 
 import * as styles from './index.module.scss';
 
 type Props = {
   title: string;
   subtitle?: string;
   icon?: ReactNode;
+  to?: To;
 };
 
-const ItemPreview = ({ title, subtitle, icon }: Props) => {
+const ItemPreview = ({ title, subtitle, icon, to }: Props) => {
```

Comment:
how about renaming `to` to `href`?
Like `TabNav`
<img width="601" alt="image" src="https://user-images.githubusercontent.com/10806653/157372042-6a25a5b0-0164-47b5-a1d4-dd6fa45ca49f.png">

Reply:
`to` is a `react-router` specific prop, i think it's good

Reply:
discussed offline